protos/feast/core/FeatureReference.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package feast.core;
+
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+option java_outer_classname = "FeatureReferenceProto";
+option java_package = "feast.proto.core";
+
+import "feast/core/Feature.proto";
+
+
+// A reference to features in a feature view
+message FeatureReference {
```

Comment:
Our existing definition of a feature reference is a reference to a single feature, not a collection. What about using `FeatureViewRef`? Or alternatively, is there a downside to using a full FeatureView object?

Reply:
I'm okay using FeatureViewRef - I don't want to use a full FeatureView object because I think we should be able to support only using a subset of features from a feature view, so we need a layer of indirection. Does that sound reasonable?

Reply:
sgtm